README.md

```diff
@@ -27,3 +27,4 @@ Copyright 2020 Goldman Sachs
 Distributed under the [Apache License, Version 2.0](http://www.apache.org/licenses/LICENSE-2.0).
 
 SPDX-License-Identifier: [Apache-2.0](https://spdx.org/licenses/Apache-2.0)
+
```

Comment:
Are there any considerations/gotchas around 
1/ Case sensitivity of database/table/column identifiers 
2/ Quoting of identifiers to support names with spaces, special chars 
3/ Special handling for database schemas etc. For e.g in Spanner, you cannot have a non default schema. So I'm thinking of throwing an error if a model has a non default schema 
4/ Any SQL compatibility issues to be tested. See testBigQueryCompatibility,pure and other pure tests under legend-pure-code-compiled-core-relational

Reply:
This PR introduces the new database type into the latest V_X__X_X version. Once this PR is merged, the changes need to also be introduced in the versioned protocols. See v_120_0 etc

Reply:
- I handled back quotes of table / column names and could not spot any special complexity to handle
- unit tested all functions
- ported changes to `v_120_0` as well